nmrih/resource/nmrih_schinese.txt

```diff
@@ -69,7 +69,7 @@
 
 "NMR_Tertiary_Attack"			"第三攻击"
 "NMR_Drop_Weapon"			"丢弃当前武器"
-"NMR_Gameplay"				"GAMEPLAY"
+"NMR_Gameplay"				"游戏设定"
```

Comment:
Similarly, I didn't find this string anywhere in the user interface
I think this may be on the server or has been abandoned, just like the game even comes with an AI mode that allows some stupid AI teammates to be generated. It may be the source engine

Reply:
> Similarly, I didn't find this string anywhere in the user interface I think this may be on the server or has been abandoned, just like the game even comes with an AI mode that allows some stupid AI teammates to be generated. It may be the source engine

I just saw that it was English, and some other languages also translated it
So I don't think it will be too big a problem, maybe?

Reply:
Yes, that's an unused string.